### DISCUSSION TITLE
Question about Launcher Background images...

### DISCUSSION THREAD

**Discussion by cabera2** [neon-nyan/Collapse#625] [Q&A] [answered]
Sorry about a question that not directly related to Collapse.
(But I think here is only place I can get the answer)

Recently I get the launcher background from
'AppData\LocalLow\CollapseLauncher\GameFolder\_img\bg'
and I remember, When there was no Hoyoplay(Official launcher), the BG image files was in each game folder.
But after Hoyoplay release, I can't fine where the Hoyoplay stores the BG images.
Does Hoyoplay still saves BG as image files? if it is, where can I find them?

**Comment by bagusnl** [comment]
Heyya~
The background files is still there, its just not given any extension

To find game background, usually you could just find the latest biggest file there is then rename them to add `.png` extension to it

**Reply by cabera2** [reply]
Can you please tell me which path should I look for?

**Reply by bagusnl** [reply]
1. Go to Collapse's settings
2. On _img then bg folder

**Reply by cabera2** [reply]
Sorry, I'm afraid you're misunderstanding my question.
I know where Collapse saves the BG images(and it already have webp extension)
I want to know is 'where does Hoyoplay saves their images'

**Reply by bagusnl** [reply, marked as answer]
oh! my bad :derp:
it uses chromium to render the background, hence the data is a bit scrambled on this folder `%appdata%\..\Roaming\Cognoshphere\HYP\1_0\fedata\Cache\Cache_Data`

You can open them with this https://www.nirsoft.net/utils/chrome_cache_view.html

Press File then Select Cache Folder (F9) to that folder

**Reply by cabera2** [reply]
Thank you, I found it!

**Comment by shatyuka** [comment]
You need this archive [HoYoPlay-Launcher-Background](https://github.com/Masterain98/HoYoPlay-Launcher-Background).

**Reply by shatyuka** [reply]
Auto generated from official API once a day.

**Reply by cabera2** [reply]
I already know that tool, but didn't know it has once a day limit. Thank you!
